README.md

```diff
@@ -50,8 +50,9 @@ Creates a new instance of the Zenfi SDK.
 |`partnerName`|String|The name of the platform which implements this code. This parameter is required to send events to the Zenfi API.|
 |`targets`|[Object]|The list of html elements and their configuration to be filled.|
 |`targets.dataKey`|String|The name of the key in the `leadInfo` object which contains the information to be filled.|
-|`targets.selector`|String|The css selector to find the element in the DOM.|
-|`targets.strategy`|String|Indicates the type of strategy to fill the DOM element. Possible values are: `value` (sets the value property of an input), `html` (Puts the content inside the HTML tag, used for `<div />`s). If no `strategy` is specified, it will try the `value` strategy by default.|
+|`targets.selector`|String \| Function|The css selector to find the element in the DOM. If a function is passed, it will receive an object with `{ dataKey, strategy, value }` properties and must return a string with the css selector.|
+|`targets.strategy`|String|Indicates the type of strategy to fill the DOM element. Possible values are: `value` (sets the value property of an input), `text` (puts the content inside the HTML tag as text, used for `<div />`s), `click` (triggers a click on the DOM element). If no `strategy` is specified, it will try the `value` strategy by default.|
```

Comment:
Creo que aquí el tipo más que un string es un enum

```suggestion
|`targets.strategy`|(value|text|click)|Indicates the type of strategy to fill the DOM element. Possible values are: `value` (sets the value property of an input), `text` (puts the content inside the HTML tag as text, used for `<div />`s), `click` (triggers a click on the DOM element). If no `strategy` is specified, it will try the `value` strategy by default.|
```

Reply:
También creo que vale la pena una columna default, para evitar que las descripciones digan  "default a tal cosa". Cuando no hay default sólo ponemos `-`.

Reply:
+1 Ya lo puse como Enum y agregué la columna de default.